README.md

```diff
@@ -63,3 +63,15 @@ If you choose to skip the `Makefile` and call `cargo` yourself, make sure to inc
     Finished dev [unoptimized + debuginfo] target(s) in 24.85 secs
 ```
 The `--release` argument is important as this enables optimizations and without optimizations you'll be looking at ~70% CPU utilization even when idle. With optimizations, the framework runs really light, 0% CPU utilization when idle and 1-2% at peak.
+
+#### Linking against musl WIP
+Use
+```
+[target.armv7-unknown-linux-musleabihf]
+linker = "arm-linux-gnueabihf-gcc"
+```
+```
+TARGET=armv7-unknown-linux-musleabihf make demo deploy-demo
+```
+so far:
+> thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ErrUnaligned', src/framebuffer/core.rs:67:23
```

Comment:
@canselcik (or anyone) any idea why mmap would fail with musl but otherwise work with glibc at https://github.com/canselcik/libremarkable/blob/a71f389e970df79ce8fba826fc658df54e4d6dbf/src/framebuffer/core.rs#L67 ?

Reply:
tried testing with using musl's mmap directly? (i. e. from c/c++ instead of through the rust mmap library/bindings)

Reply:
Hi! Thanks for dropping by :)
Turns out rustc's pre-linking optimization was resolving symbols wrong: using LTO fixes it.